packages/react-native-win32/package.json

```diff
@@ -68,14 +68,14 @@
     "just-scripts": "^0.44.7",
     "prettier": "1.19.1",
     "react": "17.0.1",
-    "react-native": "0.0.0-6fd684150",
+    "react-native": "0.0.0-c66aa8abc",
```

Comment:
>0.0.0-c66aa8ab [](start = 21, length = 14)

Why is this such a weird dependency? Shouldn't it be just a regular version?

Reply:
The versioning scheme for nightly RN builds is `0.0.0-<commitHash>`. This specific scheme has some special meaning in semver such that it can be always breaking, and is never chosen over other releases.